packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3767,11 +3743,11 @@ export class ContainerRuntime
 	 * @param summaryLogger - logger to use when fetching snapshot from storage
 	 * @returns downloaded snapshot's reference sequence number
 	 */
-	private async refreshLatestSummaryAckFromServer(
+	private async closeSummarizerOnSummaryStateStale(
 		summaryLogger: ITelemetryLoggerExt,
-	): Promise<{ latestSnapshotRefSeq: number; latestSnapshotVersionId: string | undefined }> {
+	): Promise<IBaseSummarizeResult> {
 		const readAndParseBlob = async <T>(id: string) => readAndParse<T>(this.storage, id);
-		const { versionId, latestSnapshotRefSeq } = await this.fetchSnapshotFromStorage(
```

Comment:
Probably add a comment explaining that this is to prime the cache before closing, for the likely case where this same parent is elected as summarizer again.